packages/website/src/examples/layout/Table/TableEmptyState.demo.tsx

```diff
@@ -1,18 +1,21 @@
 import {BlankSlate, Button, ColumnDef, createColumnHelper, Table, Title, useTable} from '@coveord/plasma-mantine';
 import {NoContentSize32Px} from '@coveord/plasma-react-icons';
+import {FunctionComponent} from 'react';
 
 export type Person = {
     firstName: string;
     lastName: string;
     age: number;
 };
 
-const NoData = () => {
-    const {isFiltered, clearFilters, state} = useTable();
-
-    return isFiltered ? (
+const EmptyState: FunctionComponent<{isFiltered: boolean; clearFilters: () => void; filter: string}> = ({
+    clearFilters,
+    isFiltered,
+    filter,
+}) =>
+    isFiltered ? (
```

Comment:
I think this example points in the wrong direction.
You shouldn't go from a state with a filter that gives nothing to an empty state.

I think the example should focus on the empty state.

Reply:
This example is there to show how to differenciate between an empty state for a filtered table version no data